packages/commons/src/Utility.ts

```diff
@@ -69,8 +69,22 @@ export class Utility {
     return false;
   }
 
+  public getDefaultServiceName(): string {
+    return this.defaultServiceName;
+  }
+
   public isColdStart(): boolean {
     return this.getColdStart();
   }
 
+  /**
+ * Validate that the service name provided is valid.
+ * Used internally during initialization.
+ * 
+ * @param serviceName - Service name to validate
+ */
+  public static isValidServiceName(serviceName?: string): boolean {
```

Comment:
I left this as a static method, but it's now public (previously `private`)

Reply:
I've not added functionality to the `Logger` or `Metric` class to use this method. It would be good to do. I was debating whether it should be included in this (at risk of making the PR larger), or a separate PR

Reply:
Let's try with `protected`, if it doesn't work let me know in a comment here & keep `public static`.

I would say let's do it in this PR, it should add just a few of lines extra to the diff anyway.